hack/update-codegen.sh

```diff
@@ -16,49 +16,55 @@ cd "${REPO_ROOT}"
 echo "Generating with deepcopy-gen"
 GO111MODULE=on go install k8s.io/code-generator/cmd/deepcopy-gen
 deepcopy-gen \
-    --go-header-file hack/boilerplate.go.txt \
-    --input-dirs=./pkg/apis/cluster/v1alpha2 \
+    --go-header-file ../../../../../hack/boilerplate.go.txt \
+    --input-dirs=./cluster/v1alpha2 \
     --output-file-base=zz_generated.deepcopy
 deepcopy-gen \
-    --go-header-file hack/boilerplate.go.txt \
-    --input-dirs=./pkg/apis/clusterpedia/v1beta1 \
+    --go-header-file ../../../../../hack/boilerplate.go.txt \
+    --input-dirs=./clusterpedia/v1beta1 \
     --output-file-base=zz_generated.deepcopy
 deepcopy-gen \
-    --go-header-file hack/boilerplate.go.txt \
-    --input-dirs=./pkg/apis/clusterpedia \
+    --go-header-file ../../../../../hack/boilerplate.go.txt \
```

Comment:
It is not recommended to use too many b paths in the script, the whole script is based on REPO_ROOT, it might be clearer to assign the path of the api repo to the variable and then use it in the script.
example: STAGING_REPO_PATH=${REPO_ROOT}/staging/src/github.com/clusterpedia-io

Also based on the repo directory, the following command `cd ../../../../..` can be avoided.

Reply:
done